client/src/placement-context.tsx

```diff
@@ -40,15 +40,17 @@ export interface PlacementData {
   };
 }
 
-type PlacementType = "side" | "top";
+type PlacementType = "side" | "top" | "hpMain" | "hpFooter";
 export interface PlacementContextData
   extends Partial<Record<PlacementType, PlacementData>> {
   status: Status;
 }
 
 const PLACEMENT_MAP: Record<PlacementType, RegExp> = {
   side: /\/[^/]+\/(docs\/|blog\/|search$|_homepage)/i,
```

Comment:
Unrelated to this PR, but should we be fetching the side placement on the dev `_homepage`, but not the *actual* homepage?